index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>LYNN TRICKEY</title>
+    <link href="styles/normalize.css" rel="stylesheet">
+    <link href="styles/main.css" rel="stylesheet">
+    <script src="https://use.fontawesome.com/0d65d62378.js"></script>
+  </head>
+  <body>
+    <div id="topbar-menu">
+    </div>
+
+    <header>
+      <h1><a href="index.html">She's Trickey</a></h1>
+      <nav>
+        <ul>
+          <li><a href="portfolio.html">CODE</a></li>
+          <li><a href="hobby-blog.html">COMEDY</a></li>
+          <li><a href="about.html">CONTEXT</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="top">
+        <img src="images/lynn&wesley.jpg" alt="lynn">
+        <section id="sticky-lynn">
```

Comment:
@sckirk thanks for the feedback!  I don't have the site live yet - want to do some more with it before it goes live.  Originally sticky-lynn was going to be display: sticky so it stuck to the top of the screen when people scrolled, however I trashed that idea later on as it was distracting and didn't really serve a purpose.  I probably should change the name of that id!